src/openrct2/paint/Supports.cpp

```diff
@@ -772,8 +772,8 @@ bool metal_a_supports_paint_setup(
     }
     else
     {
-        int8_t xOffset = loc_97AF20[segment].x;
-        int8_t yOffset = loc_97AF20[segment].y;
+        int8_t xOffset = SupportBoundBoxes[segment].x;
```

Comment:
We can avoid this assignment if we refactor sub_98196C to also use CoordsXY, but this should be done in another PR. Right now it accesses the memory 3 times which isn't too nice.

Reply:
What do you suggest I do right now?

Reply:
Nothing, like I stated this should be done in another PR honestly as it will change the declaration of those functions and requires a lot more modifications.

Reply:
Alright, cool